src/web/types/question-details-impl/feedback-contribution-question-details.impl.ts

```diff
@@ -18,12 +18,15 @@ import { AbstractFeedbackQuestionDetails } from './abstract-feedback-question-de
 export class FeedbackContributionQuestionDetailsImpl extends AbstractFeedbackQuestionDetails
     implements FeedbackContributionQuestionDetails {
 
-  isNotSureAllowed: boolean = true;
+  isZeroSum: boolean = true;
+  isNotSureAllowed: boolean = false
+  ;
```

Comment:
- Why is `;` on a single line?
- Same comment about the default values of the two attributes.

Reply:
Sorry, my bad!